pkg/sink/elasticsearch.go

```diff
@@ -154,13 +164,19 @@ func (e *Elasticsearch) flushIndex(ctx context.Context, indexCfg config.ELSIndex
 			},
 		}
 		_, err := e.client.CreateIndex(indexName).BodyJson(mapping).Do(ctx)
-		if err != nil {
+		if err != nil && elastic.ErrorReason(err) != elasticErrorReasonResourceAlreadyExists {
 			return fmt.Errorf("while creating index: %w", err)
 		}
 	}
 
 	// Send event to els
-	_, err = e.client.Index().Index(indexName).Type(indexCfg.Type).BodyJson(event).Do(ctx)
+	indexService := e.client.Index().Index(indexName)
+	if strings.HasPrefix(e.clusterVersion, "7.") {
```

Comment:
BTW, shouldn't we also check for versions below? E.g. 6? WDYT?

Reply:
That's a good question :) Let me add check for that

Reply:
@pkosiec also added test https://github.com/kubeshop/botkube/pull/1215/commits/3eaf7562633614c9938cd20cc10feb9b05464a11